packages/schemas/src/modeSchema.ts

```diff
@@ -0,0 +1,7 @@
+import { z } from 'zod';
+
+export const modeSchema = z.union([
+  z.literal('mode'),
```

Comment:
should mode be active ?

Reply:
yeah not sure what happened here...will update